topcoffea/cfg/mc_background_samples.cfg

```diff
@@ -3,83 +3,95 @@
 # Central UL16APV background samples
 #root://ndcms.crc.nd.edu/
 root://deepthought.crc.nd.edu/
-../../topcoffea/json/background_samples/central_UL/UL16APV_DY10to50.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_DY50.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_ST_antitop_t-channel.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_ST_top_s-channel.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_ST_top_t-channel.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_tbarW.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_TTGJets.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_TTJets.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_tW.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_WJetsToLNu.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_WWTo2L2Nu.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_WWW_4F.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_WWZ_4F.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_WZTo3LNu.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_WZZ.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_ZZTo4L.json
-../../topcoffea/json/background_samples/central_UL/UL16APV_ZZZ.json
+../../topcoffea/json/background_samples/central_UL/UL16APV_ZGToLLG.json
+../../topcoffea/json/background_samples/central_UL/UL16APV_TTTo2L2Nu.json
+../../topcoffea/json/background_samples/central_UL/UL16APV_TTToSemiLeptonic.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_DY10to50.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_DY50.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ST_antitop_t-channel.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ST_top_s-channel.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ST_top_t-channel.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_tbarW.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_TTGJets.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_TTJets.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_tW.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WJetsToLNu.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WWTo2L2Nu.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WWW_4F.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WWZ_4F.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WZTo3LNu.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WZZ.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ZZTo4L.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ZZZ.json
```

Comment:
Looks like you pushed this by accident. Can you uncomment everything?

Reply:
Yep, sorry, already done

Reply:
Thanks!